File: Vorbereitung/Übungen/2/2-1.c
int main() {
    int x;
    scanf("%d",  &x);

    // Bedingung
    if (x%2 == 0) {
        printf("Gerade");
    }
    else {
        printf("Ungerade");
    }

    //switch-case
    switch (x%2) {
        case 0:
            printf("Gerade");
            break;
        default:
            print("Ungerade");
    }
}

File: Vorbereitung/Übungen/2/2-1.cs
namespace Uebung;

class Program {
    static void Main(string [] args) {
        int x = Console.ReadLine();

        // Bedingung

        if (x%2 == 0) {
            Console.WriteLine("Gerade");
        }
        else {
            Console.WriteLine("Ungerade");
        }

        // switch-case
        string IsFlat = x%2 switch {
            0 => "Gerade",
            _ => "Ungerade"
        }

        Console.WriteLine(IsFlat);
    }
}